.travis.yml

```diff
@@ -28,6 +28,7 @@ script:
   - pip install .[tests]
   - pip install dash[testing]
   - pycodestyle webviz_subsurface/containers webviz_subsurface/datainput tests
```

Comment:
We can probably remove `pycodestyle` here as `black` is ensuring PEP8 style. Could however consider adding `pylint` (in a separate PR perhaps) since `pylint` is more than just pure style check (unused variables, unused imports etc.)

Reply:
I agree. Removed pycodestyle. Lets add pylint in a separete PR.